ReactAndroid/src/main/java/com/facebook/react/views/webview/ReactWebViewManager.java

```diff
@@ -445,6 +473,19 @@ public void setSource(WebView view, @Nullable ReadableMap source) {
     view.loadUrl(BLANK_URL);
   }
 
+  @ReactProp(name = "urlSchemeBlacklist")
+  public void setUrlSchemeBlacklist(WebView view, @Nullable ReadableArray urlSchemeBlacklistArray) {
```

Comment:
If it's from a non-null to another non-null, then the previous value will just be overwritten, right?
But, if you mean from a non-null value to a null value; yeah that would fail here. Will fix.